release-notes/README.md

```diff
@@ -4,6 +4,22 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## August 5, 2021
+
+### Android Target 1.2.7
+
+`TargetRequest` class now provides a constructor with a new `AdobeTargetDetailedCallback`. This callback, when implemented, is invoked with the Target content and a data payload map containing one or more of response tokens, analytics payload and click metric analytics payload, if available in the Target retrieve location content response with or without a prior prefetch call.
+
+**Note**: Adobe Target SDK will continue sending analytics for target payload to Analytics client-side if Adobe Analytics SDK is integrated and registered in the mobile app.
+
+### iOS Target 3.1.0
+
+* `TargetRequest` class now provides a constructor with a new `contentWithDataCallback`. This callback, when implemented, is invoked with the Target content and a data payload map containing one or more of response tokens, analytics payload and click metric analytics payload, if available in the Target retrieve location content response with or without a prior prefetch call.
+
+**Note**: Adobe Target SDK will continue sending analytics for target payload to Analytics client-side if Adobe Analytics SDK is integrated and registered in the mobile app.
+
+* Fixed an issue where click notification was not being sent to Adobe Target for a retrieved mbox location upon `clickedLocation` API call.
```

Comment:
"... where click notifications were not..."

"... upon the `clickedLocation` API call."

Reply:
1. There is a single notification request that goes to Target, isn't singular `...click notification was...` more appropriate.
2. "... upon the clickedLocation API call." - sounds good.

Reply:
Ahh, it's a singular. Then I'd go with "where the click notification was not..." instead, since without some sort of article, it sounds a bit funny. (Same goes for the second one as well.)

Reply:
haha yeah, that's why we need your valuable feedback!

Reply:
Note here that this is standard behavior. Something like:
This SDK extension, per previous behavior, will make requests to Adobe Analytics (if the Adobe Analytics extension is also implemented) with appropriate Target payloads for A4T functionality